packages/lesswrong/components/walledGarden/WalledGardenPortal.tsx

```diff
@@ -227,7 +227,7 @@ const WalledGardenPortal = ({ classes }: { classes: ClassesType }) => {
           Hide Footer
         </div>
       }
-      <GatherTownIframeWrapper  iframeRef={iframeRef}/>
+      {/*<GatherTownIframeWrapper  iframeRef={iframeRef}/>*/}
```

Comment:
Doesn't commenting this out just... break the page entirely?

Reply:
Note the WIP. I'm currently only working on the portal bar, I don't need the iframe during this development.

Reply:
Oh, I see; nvm.